src/opm/output/eclipse/AggregateWellData.cpp

```diff
@@ -304,7 +303,7 @@ namespace {
             // the target control mode requested in the simulation deck.
             // This item is supposed to be the well's actual, active target
             // control mode in the simulator.
-            iWell[Ix::ActWCtrl] = ctrlMode(well);
+            iWell[Ix::ActWCtrl] = ctrlMode(well, st);
```

Comment:
I believe this `st` argument is a left-over from an earlier edition of the patch.

Reply:
Ehh - no; the callsite you are pointing at now is actually in use.

Reply:
> Ehh - no; the callsite you are pointing at now is actually in use.

Ooops.  Yes, you're right.  I misread `ctrlMode` as `compOrder`.  Sorry.